### ISSUE TITLE
ath: phy0: DMA failed to stop

### ISSUE THREAD

**Issue by mustaqimM** [void-linux/void-packages#36635]
My wifi network has been dropping for a few versions ( I can't remember how many) with an error shown in `dmesg`:
```zsh
ath: phy0: DMA failed to stop in 10 ms AR_CR=0x00000024 AR_DIAG_SW=0x02000020 DMADBG_7=0x00006100
```
This occurs on `lts-5.10_1`, `5.15.30_1` and `mainline-5.16.16_1`, but so far not on `4.19.237_1`.

Connection is randomly lost and re-connection also occurs randomly. Restarting services like dhcpcd and wpa_supplicant (or separately iwd with dhcp), doesn't work.

### System

* xuname:  
  `Void 4.19.237_1 x86_64 GenuineIntel notuptodate rrmFFFFFFFFFFFFFFFF`

### Expected behavior
Network connection shouldn't be lost

### Actual behavior
Connection lost

### Steps to reproduce the behavior
Boot with any of the latest `5.x` versions of the kernel.

**Comment by ologantr** [comment]
Considering you wifi card is an ath one (judging by you dmesg output),
I think this is the same as #36447.
This issue was fixed in Linux versions 5.10.110, 5.15.33, 5.16.19 and 5.17.2.
None of those is in void-packages yet, though.

**Comment by mustaqimM** [comment]
Thanks so much for tracking that down, it was exactly that, even ksoftirqd's usage of a core. It seems to have been resolved with 5.16.20
